yt/utilities/on_demand_imports.py

```diff
@@ -55,7 +55,7 @@ def __init_subclass__(cls):
 
     def __new__(cls):
         if cls is OnDemand:
-            raise TypeError("The OnDemand base class cannot be instantiated.")
+            raise TypeError("The OnDemand base class cannot be instanciated.")
```

Comment:
Why was the spelling of instantiated and mimic changed?

Reply:
sorry for taking so long to reply ! this looks like some weird side effect of the large rebase operation we did on your branch, I reverted it in a fixup commit, which should hopefully get tests green again